sympy/external/tests/test_autowrap.py

```diff
@@ -58,14 +58,14 @@ def runtest_autowrap_twice(language, backend):
 
 def runtest_autowrap_trace(language, backend):
     has_module('numpy')
-    trace = autowrap(A[i, i], language, backend)
+    trace = autowrap(EinsteinSum(A[i, i]), language, backend)
```

Comment:
Changes to tests often imply backwards incompatibilities. We will likely need to support new and old functionality with deprecation warnings for the old. Can you comment on this?